src/plugins/otopi/core/log.py

```diff
@@ -203,12 +206,33 @@ def __init__(self, context):
         self._handler = None
         self._logerror = None
         self.environment[constants.CoreEnv.LOG_FILTER_KEYS] = []
+        self.environment[constants.CoreEnv.LOG_FILTER_RE] = []
+        self.environment[constants.CoreEnv.LOG_FILTER_QUESTIONS] = []
+        self.environment[constants.CoreEnv.LOG_FILTER_QUESTIONS_KEYS] = set([
+            k for k in self.environment.keys()
+            if k.startswith(constants.CoreEnv.QUESTION_PREFIX) and
+            # E.g. QUESTION/2/OVESETUP_CONFIG_ADMIN_SETUP
+            k.split('/')[-1] in self.environment[
+                constants.CoreEnv.LOG_FILTER_QUESTIONS
+            ]
+        ])
         self._filtered_keys_at_setup = []
 
     def _setupLogging(self):
         self.environment[constants.CoreEnv.LOG_FILE_HANDLE] = None
         self.environment[constants.CoreEnv.LOG_FILTER] = self._MyLoggerFilter()
-        self.environment[constants.CoreEnv.LOG_FILTER_RE] = []
```

Comment:
Why this one is removed?

Reply:
I moved it to __init__, line 209, where it made more sense (also e.g. after line 208). IIRC it was also too late here, for some reason, but I do not remember the details anymore.